docs/tutorials/api/makers.ipynb

```diff
@@ -0,0 +1,343 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Makers - Data reduction\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "\n",
+    "The `gammapy.makers` sub-package contains classes to perform data reduction tasks\n",
+    "from DL3 data to binned datasets.\n",
+    "In the data reduction step the DL3 data is prepared for modeling and fitting,\n",
+    "by binning events into a counts map and interpolating the exposure, background,\n",
+    "psf and energy dispersion on the chosen analysis geometry.\n",
+    "\n",
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gammapy.makers import MapDatasetMaker, FoVBackgroundMaker, SafeMaskMaker\n",
+    "from gammapy.datasets import MapDataset\n",
+    "from gammapy.data import DataStore\n",
+    "from gammapy.maps import MapAxis, WcsGeom, Map\n",
+    "from regions import CircleSkyRegion\n",
+    "from astropy import units as u"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dataset\n",
+    "\n",
+    "The counts, exposure,\n",
+    "background and IRF maps are bundled together in a data structure named `MapDataset`.\n",
+    "To handle on-off observations Gammapy also features a `MapDatasetOnOff` class, which\n",
+    "stores in addition the `counts_off`, `acceptance` and `acceptance_off` data.\n",
+    "\n",
+    "The first step of the data reduction is to create an empty dataset. A `MapDataset` can be created from any `WcsGeom` object. This is illustrated in the following example:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "energy_axis = MapAxis.from_bounds(\n",
+    "    1, 10, nbin=11, name=\"energy\", unit=\"TeV\", interp=\"log\"\n",
+    ")\n",
+    "geom = WcsGeom.create(\n",
+    "            skydir=(83.63, 22.01),\n",
+    "            axes=[energy_axis],\n",
+    "            width=5*u.deg,\n",
+    "            binsz=0.05*u.deg,\n",
+    "            frame='icrs'\n",
+    ")\n",
+    "dataset_empty = MapDataset.create(geom=geom)\n",
+    "print(dataset_empty)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ``.create()`` method has\n",
+    "additional options, e.g. it is also possible to specify a true energy axis:"
```

Comment:
Perhaps mention that this is the binning of the IRFs.
Also, it is possible to specify other quantities like `rad_axis`, `migra_axis` and `bins_irf`

Reply:
- added : "it is also possible to specify a true energy axis **which defines the binning of the instrument response functions**:
- for the other options I will just show show them thought  a `help(MapDataset.create)` call, as a notebook don't have to repeat all the docstring